support-frontend/assets/pages/paper-subscription-landing/components/content/helpers.jsx

```diff
@@ -34,7 +34,7 @@ export type ContentTabPropTypes = {|
   getRef: (?HTMLElement)=> void
 |};
 
-const promoTermsUrl = promotionTermsUrl(getQueryParameter(promoQueryParam) || getPromoCode('Paper', GBPCountries, 'GE19SUBS'));
+const promoTermsUrl = promotionTermsUrl(getQueryParameter(promoQueryParam) || getPromoCode('Paper', GBPCountries, 'SEP2512VHD'));
```

Comment:
And should this come out since this sale is over?

Reply:
Yes

Reply:
👍